Homework5/task2.py

```diff
@@ -0,0 +1,29 @@
+# 2. Создайте декоратор, который хранит результаты вызовов функции
+# (за все время вызовов, не только текущий запуск программы)
+
+
+def data_to_file(func):
+    def wrapper():
+        with open('file.txt', 'a') as f:
+            f.write(func() + '\n')
```

Comment:
it should return function result and keep results in file
we can add to file file name and date time